test/e2e/utils.go

```diff
@@ -141,3 +141,19 @@ func createSelfSignedMTLSArtifacts(s *v1.Secret) error {
 
 	return nil
 }
+
+func getSecurityContext() *v1.SecurityContext {
```

Comment:
Do you think this would be an improvement? I'm a bit split between just letting it be or giving it a name more meaningful 🤔 
```suggestion
func getSecurityContextRestrictedProfile() *v1.SecurityContext {
```

Reply:
That's a good idea, I'll change it to that